photon-api/src/main/scala/com/linkedin/photon/ml/algorithm/RandomEffectCoordinate.scala

```diff
@@ -134,21 +135,20 @@ object RandomEffectCoordinate {
    *
    * @note The score is the dot product of the model coefficients with the feature values (in particular, does not go
    *       through non-linear link function in logistic regression!).
-   *
    * @param randomEffectDataSet The active dataset to score
    * @param randomEffectModel The model to score the dataset with
    * @return The computed scores
    */
   protected[algorithm] def score(
     randomEffectDataSet: RandomEffectDataSet,
-    randomEffectModel: RandomEffectModel): KeyValueScore = {
+    randomEffectModel: RandomEffectModel): SimpleDataScores = {
 
     val activeScores = randomEffectDataSet
       .activeData
       .join(randomEffectModel.modelsRDD)
```

Comment:
I'm thinking - for comments like that, maybe a README in the module would be more useful: it would centralize "how it works" comments that can be useful when looking at various places in the code, whereas currently, you have to know where the comment is to even see it... :-(